install.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# init script that creates an executable in /usr/local/bin
+# so that ci_edit can be used from any directory
```

Comment:
Maybe a slight re-wording to say that it copies the files to /opt and adds a sym-link in /usr/local/bin?